perllib/FixMyStreet/App/Form/Waste/Bulky.pm

```diff
@@ -34,9 +34,15 @@ has_page cannot_book => (
 
 has_page about_you => (
     intro => 'bulky/about_you_preamble.html',
-    fields => ['name', 'email', 'phone', 'continue'],
+    fields => ['name', 'email', 'phone', 'extra_bulky_text_updates', 'continue'],
     title => 'About you',
     next => 'choose_date_earlier',
+    update_field_list => sub {
+        my $form = shift;
+        if ($form->c->cobrand->moniker eq 'peterborough') {
```

Comment:
It would be better to have a `COBRAND_FEATURES` flag for the new feature rather than hardcoding the `peterborough` check everywhere. This would make it easier to roll out to other cobrands later and also means we can turn it on/off for Peterborough with a redeploy rather than code changes.